test-jest/polar/Pie.spec.tsx

```diff
@@ -352,83 +374,110 @@ describe('<Pie />', () => {
           onMouseEnter={onMouseEnter}
           onMouseLeave={onMouseLeave}
           onClick={onClick}
+          dataKey="cy"
         />
       </Surface>,
     );
-    const se = wrapper.find(Layer).at(3);
-
-    se.simulate('mouseEnter');
-    expect(onMouseEnter.calledOnce).to.equal(true);
-    se.simulate('mouseLeave');
-    expect(onMouseLeave.calledOnce).to.equal(true);
-    se.simulate('click');
-    expect(onClick.calledOnce).to.equal(true);
+
+    const se = container.querySelectorAll('.recharts-layer')[4];
+    await userEvent.hover(se);
+    await waitFor(() => {
+      expect(onMouseEnter).toHaveBeenCalledTimes(1);
+    });
+    await userEvent.unhover(se);
+    await waitFor(() => {
+      expect(onMouseLeave).toHaveBeenCalledTimes(1);
+    });
+    await userEvent.click(se);
+    await waitFor(() => {
+      expect(onClick).toHaveBeenCalledTimes(1);
+    });
   });
 
-  it('Handles keyboard interaction: Tab can focus in and out of the pie chart', async () => {
+  test('Handles keyboard interaction: Tab can focus in and out of the pie chart', async () => {
+    expect.assertions(3);
     const timeout = 2000;
-    const { container } = testingLibraryRender(
+    const { container } = render(
       // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
       <div tabIndex={0} className="container">
         <Surface width={500} height={500}>
-          <Pie isAnimationActive={false} cx={250} cy={250} label innerRadius={0} outerRadius={200} sectors={sectors} />
+          <Pie
+            isAnimationActive={false}
+            cx={250}
+            cy={250}
+            label
+            innerRadius={0}
+            outerRadius={200}
+            sectors={sectors}
+            dataKey="cy"
+          />
         </Surface>
       </div>,
     );
     const pie = container.getElementsByClassName('recharts-pie')[0];
     const pieContainer = document.getElementsByClassName('container')[0];
 
-    pieContainer.focus();
+    (pieContainer as any).focus();
```

Comment:
any way this doesn't have to be `any`?

Reply:
I've casted the type to an `HTMLElement` to preserve typing.